doc/getting-started-with-docker.md

```diff
@@ -0,0 +1,177 @@
+# Getting Started With Docker
+
+This section describes how to run device-onvif-camera with docker.
+
+## 1. Build docker image
+Build docker image named edgex/device-onvif-camera:0.0.0-dev with the following command:
+```shell
+make docker
+```
+
+## 2. Prepare edgex-compose/compose-builder
+Download the [edgex-compose](https://github.com/edgexfoundry/edgex-compose) and setup it according to the [docker-compose setup guide](./docker-compose/README.md)
+
+## 3. Deploy EdgeX services and device-onvif-camera
+ 1. Change directory to the `edgex-compose/compose-builder`
+ 2. Deploy services with the following command:
+```shell
+make run no-secty device-onvif-camera
```

Comment:
I think should be "ds-onvif-camera", based on prior review feedback.

Reply:
Modified the command.